CodeBeam.MudExtensions/Components/Loading/MudLoading.razor

```diff
@@ -30,8 +30,8 @@
         else
         {
 
-            <MudProgressLinear Class="mb-2" Color="Color.Primary" Indeterminate="true" />
-            <MudText Class="@TextClassname" Align="Align.Center">@Text</MudText>
+            <MudProgressLinear Class="@LoaderClassname" Style="@LoaderStylename" Color="Color.Primary" Indeterminate="true" />
```

Comment:
We can make it simpler, LoaderClassname does not have predefined class.

Can you change this like `Class="@ClassLoader"` and the other similar?

Reply:
I wanted to keep the "mb-2" class on Linear LoaderType but it diesn't seem necessary, i'll change it